meta-schema.graphql

```diff
@@ -0,0 +1,94 @@
+# Base types
+type Identifiers {
+	id:ID!
```

Comment:
What is the 'ID'  here ? the IPI/ISWC/Grid value ? if so ..it should be string?!

Should we also have a meta-id field also here ?

Reply:
> What is the 'ID' here ? the IPI/ISWC/Grid value
Look one line below, it clearly states example `idTypes`, one of which is an IPI.

> it should be string?!
http://graphql.org/learn/schema/#scalar-types
> ID: The ID scalar type represents a unique identifier, often used to refetch an object or as the key for a cache. The ID type is serialized in the same way as a String; however, defining it as an ID signifies that it is not intended to be human‐readable.